### PR TITLE
Fix coding style

### PR DESCRIPTION
**Pull request by RazvanRzv**

### CODE REVIEW

File: wouso/games/challenge/forms.py [hunk 1]
```diff
@@ -6,7 +6,8 @@ def __init__(self, challenge, data=None):
         super(ChallengeForm, self).__init__(data)
 
         for q in challenge.questions.all():
-            field = forms.MultipleChoiceField(widget=forms.CheckboxSelectMultiple, label=q)
+            CSM = forms.CheckboxSelectMultiple
```
Comment (razvanch) [inline wouso/games/challenge/forms.py]: @RazvanRzv, you are adding an unnecessary static member to the class. In order to shorten the line, you could just put `label=q` on the next line.

File: wouso/games/challenge/forms.py [hunk 2]
```diff
@@ -22,11 +23,10 @@ def get_response(self):
         adding an empty list to dic in this case"""
         for field in self.visible_fields():
             id = int(field.html_name[len('answer_'):])
-            if not res.has_key(id):
+            if not id in res:
```
Comment (razvanch) [inline wouso/games/challenge/forms.py]: @RazvanRzv, you can even write `if id not in res:`

File: wouso/games/challenge/forms.py [hunk 3]
```diff
@@ -34,13 +34,14 @@ def check_self_boxes(self):
...
     def get_results_in_order(self, results):
         """ Assign question feedback with according question """
         v = []
         for field in self.visible_fields():
             id = int(field.html_name[len('answer_'):])
-            v.append( results[id] )
-        return v
-
+            v.append(results[id])
+        return v
```
Comment (razvanch) [inline wouso/games/challenge/forms.py]: @RazvanRzv, always leave an empty line at the end of the file

File: wouso/games/challenge/forms.py (hunk 1 above)
Comment (RazvanRzv) [inline wouso/games/challenge/forms.py]: @razvanch  if i put label = q on the next line, the line will still have more than 80 characters.

File: wouso/games/challenge/forms.py (hunk 3 above)
Comment (RazvanRzv) [inline wouso/games/challenge/forms.py]: "forms.py:46:1: W391 blank line at end of file"
i just did what the issues was saying.

**Pull request merged**